index.html

```diff
@@ -81,27 +81,29 @@
         </div>
       </section>
       <section class="vis">
-        <div class="vis-header">
-          <!-- @todo update filter with current filter state -->
-          <h1 id="filter">Recent Site</h1>
-          <!-- @todo update view with current view state -->
-          <h2><span id="view">Graph</span> View</h2>
-          <!-- @todo display list or graph view, update view based on filter and controls -->
-          <!-- @todo remove the hard-coded width, height values -->
-        </div>
-        <div class="vis-controls info-panel-controls">
-          <!-- @todo check purpose of website icon -->
-          <button class="info-panel website"></button>
-          <!-- @todo add help sidebar -->
-          <button class="info-panel help"></button>
-          <!-- @todo add about sidebar -->
-          <button class="info-panel about"></button>
-        </div>
-        <div class="vis-content" id="visualization">
+        <div class="vis-header-wrapper">

```

Comment:
Agreed do we need `display: contents;` or something? Sometimes it is unavoidable though (haven't actually looked). I think only ff works with display contents.